src/mixins/enum-mixin.ts

```diff
@@ -110,6 +111,15 @@ export default class EnumMixin extends Vue {
     return (item.paymentMethod === PaymentMethod.ONLINE_BANKING)
   }
 
+  //
+  // Effect of Order helpers
+  //
+
+  /** Returns True if effect of order is Plan of Arrangement. */
+  isEffectOfOrderPlanOfArrangement (item: any): boolean {
```

Comment:
Would `CourtOrderIF` work here opposed to `any`?

Reply:
(Actually I don't mind `any` here as it means you can pass any object type to check if it's a match. After all this is in a general mixin. Thoughts?)

Reply:
But it references a property specific to the `CourtOrder` object, `effectOfOrder`? 
I agree with the mixin being general but this feels like a pretty specific check vs a specific variable. Perhaps this should be a getter since it is specific?

Reply:
Good points.

I don't have a preference whether this is a "enum mixin" or a "getter" (a la "isBcomp"). If we leave it leave it here then it's a bit overkill (but extensible), whereas being a getter seems over-optimized. 🤷

Reply:
haha totally, not a hill i'd like to die on 😆 
This is g2g!